server/.gitignore

```diff
@@ -10,6 +10,7 @@ Pipfile.lock
 *.dyn_o
 *.dyn_hi
 .virtualenv
+.python-venv
```

Comment:
It doesn't seem super important but I was wondering if we should standardize on a single name for python venvs (`.python-venv` seems good)

Reply:
I don’t have a strong opinion about the particular name, so anything works for me, really. I just put the name I was using in the `.gitignore` file. We could use `.virtualenv` instead, or something else, so let me know if you’d prefer something different!